### ISSUE TITLE
"status-update" messages sends an empty status on trx disconnect

### ISSUE THREAD

**Issue by gx1400** [hb9ssb/trx-control#6]
When listening for status-update packets, and trx disconnects (like trx being powered off), the status message is empty.

```Received message from server: {"request":"status-update","from":"ft-817","status":[]}```

Recommend status of "disconnected" or something similar.

**Comment by mbalmer** [comment]
Yes, this, and make the list of destinations dynamic.  Inform clients of new or disappearing devices.
